tests/review-pr-phase3-ci.test.sh

```
@@ -213,6 +215,40 @@ assert_no_grep "$REVIEW_PR" 'CI_CLASSIFICATION_PATH="\$RESEARCH_DIR_ABS/ci-class
+required=json.load(open(sys.argv[1],encoding="utf-8"))["edges"]["review_pr.ci.classify"]["required_inputs"]
+expected={
+    "pr_number":"integer",
+    "run_id":"string",
+    "head_sha":"string",
+    "log_content":"bounded_text",
+    "log_sha256":"string",
+}
+if required!=expected or "log_path" in required:
+    raise SystemExit("classifier authority is not closed over identity and bytes")
+PY
+if [ "$?" -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)